queue/index.js

```diff
@@ -0,0 +1,122 @@
+export let queueChannel = {
+  processing: false,
+  data: []
+}
+
+export class Queue {
+  constructor(app, queues, key) {
+    this.app = app
+    this.queues = queues
+    this.key = key
+    this.clientId = undefined
+  }
+
+  setClientId(clientId) {
```

Comment:
Why do need to change client ID in runtime? Will it be more readable to have immutable client ID and create new `Queue` instance for every client?

Reply:
Problem is `ServerClient` also starts with client id as `undefined` (https://github.com/logux/server/blob/main/server-client/index.js#L21). Right now queue gets client id from its respective client in `ServerClient.auth` method (https://github.com/muddv/logux-server/blob/queues/server-client/index.js#L109). Is this problematic?